### ISSUE TITLE
[use-cases] it should be clear in §3 that considering accessibility is more about human practice/policy than software

### ISSUE THREAD

**Issue by cookiecrook** [w3c/silver#669]
# End of §3: 
Final sentence is: "In this document we intentionally presume that content providers are doing their best to consider accessibility to the extent they reasonably can."

Consider adding something indicating that "their best" is not just technical means such as automation and software development, but includes company-internal policy/practices/oversight, stakeholder education/training for employees and non-employee content creators, and other methods.
